vite.config.ts

```diff
@@ -2,5 +2,10 @@ import { sveltekit } from '@sveltejs/kit/vite';
 import { defineConfig } from 'vite';
 
 export default defineConfig({
-	plugins: [sveltekit()]
+	plugins: [sveltekit()],
+	server: {
+		fs: {
+			allow: ['..'],
+		}
+	}
```

Comment:
I have no idea what this is. Can you explain?

Reply:
Required for the svg search to display. https://vitejs.dev/config/server-options.html#server-fs-allow